coreum/assetlist.json

```diff
@@ -93,6 +93,176 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/xrpl/images/xrp.svg"
       },
       "type_asset": "sdk.coin"
+    },
+    {
+      "description": "SOLO bridged from XRPL",
+      "denom_units": [
+        {
+          "denom": "xrpl11278ecf9e-core1zhs909jp9yktml6qqx9f0ptcq2xnhhj99cja03j3lfcsp2pgm86studdrz",
+          "exponent": 0,
+          "aliases": [
+            "xrpl11278ecf9e"
+          ]
+        },
+        {
+          "denom": "solo",
+          "exponent": 15
+        }
+      ],
+      "base": "xrpl11278ecf9e-core1zhs909jp9yktml6qqx9f0ptcq2xnhhj99cja03j3lfcsp2pgm86studdrz",
+      "name": "Solo",
+      "display": "solo",
+      "symbol": "SOLO",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "xrpl",
+            "base_denom": "xrpl11278ecf9e"
+          },
+          "provider": "Coreum"
+        }
+      ],
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "xrpl",
+            "base_denom": "xrpl11278ecf9e"
+          },
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/xrpl/images/solo.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/xrpl/images/solo.svg",
+          "theme": {
+            "primary_color_hex": "#ffffff"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/xrpl/images/solo.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/xrpl/images/solo.svg"
+      },
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "RLUSD bridged from XRPL",
+      "denom_units": [
+        {
+          "denom": "xrpl570c00a604-core1zhs909jp9yktml6qqx9f0ptcq2xnhhj99cja03j3lfcsp2pgm86studdrz",
+          "exponent": 0,
+          "aliases": [
+            "xrpl570c00a604"
+          ]
+        },
+        {
+          "denom": "rlusd",
+          "exponent": 15
+        }
+      ],
+      "base": "xrpl570c00a604-core1zhs909jp9yktml6qqx9f0ptcq2xnhhj99cja03j3lfcsp2pgm86studdrz",
+      "name": "RLUSD",
+      "display": "rlusd",
+      "symbol": "RLUSD",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "xrpl",
+            "base_denom": "xrpl570c00a604"
```

Comment:
This base_denom ust be registered on XRP-L's Assetlist (at /_non-cosmos/xrpl/assetlist.json)

Reply:
Fixed